client/my-sites/people/people-add-subscribers/index.jsx

```diff
@@ -0,0 +1,96 @@
+import { Card } from '@automattic/components';
+import { AddSubscriberForm } from '@automattic/subscriber';
+import { localize } from 'i18n-calypso';
+import { get } from 'lodash';
+import page from 'page';
+import PropTypes from 'prop-types';
+import { PureComponent } from 'react';
+import { connect } from 'react-redux';
+import EmailVerificationGate from 'calypso/components/email-verification/email-verification-gate';
+import EmptyContent from 'calypso/components/empty-content';
+import HeaderCake from 'calypso/components/header-cake';
+import Main from 'calypso/components/main';
+import PageViewTracker from 'calypso/lib/analytics/page-view-tracker';
+import {
+	isRequestingInvitesForSite,
+	getPendingInvitesForSite,
+	getAcceptedInvitesForSite,
+	getNumberOfInvitesFoundForSite,
+	isDeletingAnyInvite,
+} from 'calypso/state/invites/selectors';
+import { canCurrentUser } from 'calypso/state/selectors/can-current-user';
+import isPrivateSite from 'calypso/state/selectors/is-private-site';
+import { isJetpackSite } from 'calypso/state/sites/selectors';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+
+class PeopleInvites extends PureComponent {
+	static propTypes = {
+		site: PropTypes.object,
+	};
+
+	goBack = () => {
+		const siteSlug = get( this.props, 'site.slug' );
+		const fallback = siteSlug ? '/people/team/' + siteSlug : '/people/team';
+
+		// Go back to last route with /people/team/$site as the fallback
+		page.back( fallback );
+	};
+
+	render() {
+		const { site, canViewPeople, translate } = this.props;
+		const siteId = site && site.ID;
+
+		if ( siteId && ! canViewPeople ) {
+			return (
+				<Main>
+					<PageViewTracker
+						path="/people/add-subscribers/:site_id"
+						title="People > Add Subscribers"
+					/>
+					<EmptyContent
+						title={ translate( 'You are not authorized to view this page' ) }
+						illustration={ '/calypso/images/illustrations/illustration-404.svg' }
+					/>
+				</Main>
+			);
+		}
+
+		return (
+			<Main>
+				<PageViewTracker path="/people/add-subscribers/:site_id" title="People > Add Subscribers" />
+				<HeaderCake isCompact onClick={ this.goBack }>
+					{ translate( 'Add Subscribers to %(sitename)s', {
```

Comment:
Subscribers should probably be lower case

Reply:
Done ✔️